cli/tauri.js/src/api/cli.ts

```diff
@@ -4,16 +4,25 @@ interface Args {
   [key: string]: string | Object
 }
 
-function runCliCommand(
-  command: string,
-  args: Args
-): { pid: number; promise: Promise<void> } {
+interface Cmd {
+  pid: number
+  promise: Promise<void>
+}
+
+function toKebabCase(value: string): string {
+  return value
+    .replace(/([a-z])([A-Z])/g, '$1-$2')
```

Comment:
Is it possible to end with numbers so we would need `[a-z0-9]`?

Reply:
nah this is fine for now